### ISSUE TITLE
Are glottal consonants actually sonorant?

### ISSUE THREAD

**Issue by danmysak** [dmort27/panphon#30]
The glottal stop (`ʔ`) as well as both voiced (`ɦ`) and voiceless (`h`) glottal fricatives are classified as sonorant. Should they really be?

**Comment by dmort27** [comment]
Yes, this has been the general consensus among phonologists for some time.

**Comment by danmysak** [comment]
I see, thanks.
